Python/compile.c

```diff
@@ -7243,7 +7242,6 @@ makecode(struct compiler *c, struct assembler *a, PyObject *constslist,
     PyObject *consts = NULL;
     PyObject *localsplusnames = NULL;
     PyObject *localspluskinds = NULL;
-    PyObject *name = NULL;
```

Comment:
This was unused but was not reported by the compiler, probably because of the `Py_XDECREF(name)` below